content/blog/Sawan_blog.md

```diff
@@ -0,0 +1,59 @@
+---
+title: My Experience at FOSS Overflow
+author: Sawan Bhattacharya
+date: '2022-01-21'
+---
+
+
+</br>
+
+This blog is about my experience at [FOSS Overflow](https://fossoverflow.dev/) and working on [Activity Tracker](https://github.com/OpenLake/Activity-Tracker)
+
+</br>
+
+## What is FOSS Overflow?
+
+</br>
+
+<div align="center">
+
+![Alt text](https://fossoverflow.dev/nextimg/%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png/828/75?url=%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png&w=828&q=75)
```

Comment:
This image URL may break. Refer https://github.com/OpenLake/FOSS-Overflow/pull/2/files for including images.